couchdb/README.md

```diff
@@ -164,63 +164,67 @@ required options to set:
-| `placementConfig.enabled`            | false                                                                                                                                                        |
-| `placementConfig.image.repository`   | caligrafix/couchdb-autoscaler-placement-manager                                                                                                              |
-| `placementConfig.image.tag`          | 0.1.0                                                                                                                                                        |
+| Parameter                              | Default                                          |
+| -------------------------------------- | ------------------------------------------------ |
+| `adminUsername`                        | admin                                            |
+| `adminPassword`                        | auto-generated                                   |
+| `adminHash`                            |                                                  |
+| `cookieAuthSecret`                     | auto-generated                                   |
+| `image.repository`                     | couchdb                                          |
+| `image.tag`                            | 3.2.1                                            |
+| `image.pullPolicy`                     | IfNotPresent                                     |
+| `searchImage.repository`               | kocolosk/couchdb-search                          |
+| `searchImage.tag`                      | 0.1.0                                            |
+| `searchImage.pullPolicy`               | IfNotPresent                                     |
+| `initImage.repository`                 | busybox                                          |
+| `initImage.tag`                        | latest                                           |
+| `initImage.pullPolicy`                 | Always                                           |
+| `ingress.enabled`                      | false                                            |
+| `ingress.hosts`                        | chart-example.local                              |
+| `ingress.annotations`                  |                                                  |
+| `ingress.path`                         | /                                                |
+| `ingress.tls`                          |                                                  |
+| `persistentVolume.accessModes`         | ReadWriteOnce                                    |
+| `persistentVolume.storageClass`        | Default for the Kube cluster                     |
+| `persistentVolume.annotations`         | {}                                               |
+| `persistentVolume.existing`            | false                                            |
```

Comment:
Unfortunately [this PR](https://github.com/apache/couchdb-helm/pull/106) added unnecessary and excessive spacing to a table in the README, in addition to obscuring the diff.  The changes in this PR are the additions of lines #190-193.